server/src/dev/resources/application-mirror.yml

```diff
@@ -6,8 +6,10 @@ ovsx:
       requests-per-second: 5
       user-name: mirror_user
       schedule:
-        extensions: '* * * * * ? *'
-        metadata: '* * * * * ? *'
+        extensions: '*/10 * * * * ? *'
+        metadata: '*/20 * * * * ? *'
+        setActive: '*/20 * * * * ? *'
+        namespaceVerify: '*/20 * * * * ? *'
```

Comment:
can we just move active and NamespaceVerified into the mirror metadata instead of creating new schedules?